homeassistant/components/unifiprotect/binary_sensor.py

```diff
@@ -332,14 +332,19 @@ class ProtectBinaryEventEntityDescription(
     ),
 )
 
-SENSE_SENSORS: tuple[ProtectBinaryEntityDescription, ...] = (
+# The mountable sensors can be remounted at run-time which
+# means they can change their device class at run-time.
+MOUNTABLE_SENSE_SENSORS: tuple[ProtectBinaryEntityDescription, ...] = (
     ProtectBinaryEntityDescription(
         key=_KEY_DOOR,
         name="Contact",
         device_class=BinarySensorDeviceClass.DOOR,
```

Comment:
Should we remove it from the entity descriptions? It doesn't serve any purpose code wise anymore

Reply:
There is likely to be more of them in the future so probably best to leave it as-is